Rewrite JS3 week 1 prep (#351)
CodeYourFuture/curriculum: content/js3/blocks/one-to-one/index.md

+++
title = 'One to one'
headless = true
time = 30
facilitation = false
emoji= '🧩'
[objectives]
    1='Define a one to one mapping'
+++

> At the moment, we have a one to one mapping between the array of data and the UI elements. We can take a group of UI elements and

Reviewer: ```suggestion
At the moment, we have a one to one mapping between the array of data and the UI elements. Each item in the array matches a node in the UI.
```
